resources/documentation/deploy_bluebanquise.rst

```diff
@@ -81,13 +81,21 @@ Then start your main interface manually. Here enp0s3:
 
   ifup enp0s3
 
+Or on RHEL/CentOS 8:
+
+.. code-block:: bash
+
+  nmcli con up enp0s3
+
 Once interface is up (check using *ip a* command), setup the BlueBanquise
 controller:
 
 .. code-block:: bash
 
   ansible-playbook /etc/bluebanquise/playbooks/managements.yml --limit management1 --tags bluebanquise
 
+This will install the needed few rpms (python filters).
```

Comment:
```suggestion
This will install the requirements to run BlueBanquise (mostly python filters
for Ansible).
```

Not sure the detail about python filters is of interest?